apps/wallet/src/ui/app/hooks/useTransactionData.ts

```diff
@@ -14,9 +14,10 @@ export function useTransactionData(
     transaction?: TransactionBlock | null
 ) {
     const rpc = useRpcClient();
-    return useQuery(
-        ['transaction-data', transaction?.serialize()],
-        async () => {
+    return useQuery({
+        // eslint-disable-next-line @tanstack/query/exhaustive-deps
```

Comment:
I guess cases like this are just a limitation of react-query not supporting non-serializable keys / the linter not being smart enough to pick up that we're specifying a proper dependency?

Reply:
Yeah, I was going to play around with better ways to do this later, it's unfortunate that we don't have a good way to do this.